docs/service-manifest.md

```diff
@@ -346,3 +347,22 @@ spec:
       memory: 800Mi
 
 ```
+### نمونه مانیفست برای ManagedService روی dedicated-volume
```

Comment:
I think this example might mislead the users, as they need to know first about the dedicated volumes which is relatively an advance feature.

Reply:
Yes, I added another PR for dedicated volume which will not have this misleading possibility.
https://github.com/fandoghpaas/fandogh-docs/pull/66/files#diff-f3990fa0344c69c6de9ab092d0da2f25R138